.github/workflows/test_runner.yaml

```diff
@@ -19,6 +23,11 @@ jobs:
         pip install -r dev_requirements.txt
     - name: Run tests
       run: python -m pytest
+    - name: Install opensafely
+      run: pip install opensafely
+    - name: Run test suite with opensafely
+      run: opensafely exec python:${{ matrix.python-docker-image-version }}
```

Comment:
Does this need to have `pytest` on the end?

ATM, I think its trying run the python REPL, which will exit immediately as there is no stdin?